app/views/shared/_local_authority_details.html.erb

```diff
@@ -13,6 +13,20 @@
     <span class="<%= authority.label_status_class %> text-muted"><b><%= authority.homepage_status %></b></span>
     <span class="text-muted"><%= authority.homepage_link_last_checked %></span>
 
+    <div>
+      <h3>Status</h3>
+      <div>Current status: <%= authority.active? ? 'active' : 'inactive' %></div>
+      <% unless authority.active? %>
+        <div>Date authority became inactive: <%= authority.active_end_date %></div>
+        <div>Inactive reason: <%= authority.active_note %></div>
```

Comment:
The inactive label is displayed whenever an authority is inactive. Should we also only display the label and note if an `active_note` has been added?